src/Logic/Camera.java

```diff
@@ -1,27 +1,33 @@
+package Logic;
```

Comment:
[Fix] pachetele in java se scriu cu litera mica, doar clasele sunt cu PascalCase

Reply:
ok